src/vs/workbench/contrib/interactive/browser/interactiveEditor.ts

```diff
@@ -152,9 +152,11 @@ export class InteractiveEditor extends EditorPane {
 		codeEditorService.registerDecorationType('interactive-decoration', DECORATION_KEY, {});
 		this._register(this.#keybindingService.onDidUpdateKeybindings(this.#updateInputDecoration, this));
 		this._register(this.#notebookExecutionStateService.onDidChangeCellExecution((e) => {
-			const cell = this.#notebookWidget.value?.getCellByHandle(e.cellHandle);
-			if (cell && e.changed?.state) {
-				this.#scrollIfNecessary(cell);
+			if (e.notebook.toString() === this.#notebookWidget.value?.viewModel?.notebookDocument.uri.toString()) {
```

Comment:
It's safe touse `isEqual` from `vs/base/common/resources` here to compare the notebook Uri.

Reply:
Will change. Thanks.